operators/tackle-operator/2.0.0-beta.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,15 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: tackle-operator
+  operators.operatorframework.io.bundle.channels.v1: v2.0.0-beta
+  operators.operatorframework.io.bundle.channel.default.v1: v2.0.0-beta
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.15.0+git
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: ansible.sdk.operatorframework.io/v1
+
+  # Annotations for testing.
+  operators.operatorframework.io.test.mediatype.v1: scorecard+v1
+  operators.operatorframework.io.test.config.v1: tests/scorecard/
```

Comment:
Please remove these 2 lines since there is no `tests/scorecard/` in bundle

Reply:
Removed